cymetric/filters.py

```diff
@@ -0,0 +1,247 @@
+import pandas as pd
+import numpy as np
+
+try:
+    from graphviz import Digraph
+    HAVE_GRAPHVIZ = True
+except ImportError:
+    HAVE_GRAPHVIZ = False
+
+try:
+    from pyne import data
+    import pyne.enrichment as enr
+    from pyne import nucname
+    HAVE_PYNE = True
+except ImportError:
+    HAVE_PYNE = False
+
+
+import cymetric as cym
+from cymetric import tools
+from cymetric.tools import format_nuc
+from cymetric.tools import reduce
+from cymetric.tools import merge
+from cymetric.tools import add_missing_time_step
+
+
+def get_transaction_df(evaler, senders=(), receivers=(), commodities=()):
```

Comment:
In general you shouldn't include the datatype in variable or function names unless necessary.  Thus all of the `_df`s in this and the following functions should be removed, in my opinion.  Additionally, I would say that the use of abbreviations should be avoided in APIs. It is better for the interface to spell things out so that the user can use abbreviations in their code.  

Is there any reason to not just call this function `transactions()`?